spec/takeaway_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'takeaway'
+
+describe Takeaway do  
+
+	subject(:takeaway) {described_class.new(menu: menu, order: order, sms: sms)} #injecting menu class to allow amy menu methods to be handled in menu class (Order doesn't need to know!) - ## ?? ## Role of value menu?  Doesn't work without, but unsure of role...
+	
+	let(:menu) {double(:menu, print: printed_menu)} #verifying doubles...?!  skipped this step (for now...)
+	let(:order) {double(:order, total: 15.50)}  #why hardwiring number in here?!
+	let(:sms) {double(:sms, deliver: nil)}
+	let(:printed_menu) {"chicken: £3.50"}
+	let(:dishes) {{chicken: 2, fish: 1}}
+
+	it {is_expected.to respond_to(:print_menu)} #instance of Order to respond to Order's print_menu method.
```

Comment:
Metrics/LineLength: Line is too long. [104/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)